### DISCUSSION TITLE
How to get quarto cli shell completion?

### DISCUSSION THREAD

**Discussion by massisenergy** [quarto-dev/quarto-cli#3216] [Q&A]
I'm on MacOS, using quarto cli. However I cannot figure out how to enable shell completion (pressing TAB in `zsh`). I have two versions of quarto installed in my machine: one came bundled with the Studio.app, another installed using `brew install quarto`. 

For newcomers to quarto, a shell completion would be imperative to learn it quicker!

Thanks for your input.

**Comment by mcanouil** [comment]
Indeed, I have no auto-completion either.
I am using a lot `--help`/`-h`.

**Comment by mcanouil** [comment]
Non official static completions for Quarto CLI 1.9 and above supporting: bash, zsh, fish, and PowerShell: https://m.canouil.dev/quarto-completions/
